practical/bash/import_as_create_annotation.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+# -----------------------------------------------------------------------------
+#   Copyright (C) 2018 University of Dundee. All rights reserved.
+#
+#   This program is free software; you can redistribute it and/or modify
+#   it under the terms of the GNU General Public License as published by
+#   the Free Software Foundation; either version 2 of the License, or
+#   (at your option) any later version.
+#   This program is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#   GNU General Public License for more details.
+#
+#   You should have received a copy of the GNU General Public License along
+#   with this program; if not, write to the Free Software Foundation, Inc.,
+#   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# ------------------------------------------------------------------------------
+#
+# In this script we:
+#   * log in as another user
+#   * create a new dataset
+#   * import an image
+#   * upload an attachment
+#
+#
+dir=$(pwd -P)
+OMEROPATH=${OMEROPATH:-/opt/omero/server/OMERO.server/bin/omero}
+PASSWORD=${PASSWORD:-ome}
+HOST=${HOST:-outreach.openmicroscopy.org}
+SUDOER=${SUDOER:-trainer-1}
+OMEUSER=${OMEUSER:-user-1}
+IMAGEPATH=${IMAGEPATH:-}
+ATTACHMENTPATH=${ATTACHMENTPATH:-}
+
+deleteimage=false
+deletefile=false
+
+if [ -z "$IMAGEPATH" ]; then
+    touch image_to_import.fake
+    IMAGEPATH=$dir/image_to_import.fake
+    deleteimage=true
+fi
+
+if [ -z "$ATTACHMENTPATH" ]; then
+    touch file_to_upload.csv
+    ATTACHMENTPATH=$dir/file_to_upload.csv
+    deletefile=true
+fi
+
+# Log in as another user
+$OMEROPATH login --sudo ${SUDOER} -u $OMEUSER -s $HOST -w $PASSWORD
+
+# Create a dataset as the specified user
+dataset=`$OMEROPATH obj new Dataset name='Basel-workflow'`
+
+# Import the image in the newly created dataset
+result=`$OMEROPATH import -T $dataset $IMAGEPATH --output ids`
+
+# Retrieve the Image's id. It is returned as Image:123
+imageid=`cut -d':' -f2 <<< $result`
```

Comment:
I guess it is this line which fails with 

``ValueError: invalid literal for long() with base 10: '29320,29321,29322'``

if the image ID returned is not just one number (the svs which I imported consists of three images in OMERO).

I think there could be 2 solutions:

   - parse the multi-ID output, and select just the first ID, and add the attachment to that image
   - parse the ID output only so far as to ascertain whether or not there are multiple IDs returned. If multiple IDs are returned, then inform the user "sorry, multiple images imported, do not know to which attach the file" and stop the script there